02LessonTwo/in-class-project/index.html

```diff
@@ -1,10 +1,49 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  <title>The Sweet Spot</title>
+  <link type="text/css" rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <img src="http://images.smh.com.au/2012/07/20/3471309/cupcake_girl_729_410-620x349.jpg" class="center"/>
+  <h1 style="color:pink" "text-align:center">The Sweet Spot</h1>
+  <h3 class="center"><em>Best Baked Goods in Town</em></h3>
```

Comment:
forgot to define the class of 'center' in CSS. otherwise, this is fantastic code. great formatting.